pycheribuild/projects/cross/gdb.py

```diff
@@ -80,8 +80,8 @@ class BuildGDBBase(CrossCompileAutotoolsProject):
     @classmethod
     def dependencies(cls, config: CheriConfig) -> "list[str]":
         deps = super().dependencies(config)
-        # For the native-hybrid build gmp must be installed via ports.
-        if not cls.get_crosscompile_target().is_cheri_hybrid():
+        # For the native and native-hybrid builds gmp must be installed via ports.
+        if not cls.get_crosscompile_target().is_native():
```

Comment:
I guess this is fine too, we can require gmp to use packages on Linux and macos as well easier than bootstrapping it by default.